Ansible/roles/marvin/templates/test_data.py.j2

```diff
@@ -200,6 +204,14 @@ test_data = {
         "endip": "172.16.15.41",
         "acltype": "Account"
     },
+    "l2-network_offering": {
+        "name": 'Test L2 - Network offering',
```

Comment:
@borisstoyanov you may want to fix the single quotes `'` with double quotes `"` to be consistent with how we define string in test_data, however for a python dictionary it may not cause any issues.

Reply:
fixed